lint check fails on master

A recent change of flake8 or its config causes the code style check to fail on the master branch - it complains about a ambiguous variable name in few files:

tests/unit/v1/test_order.py:210:16: E741 ambiguous variable name 'l'
tests/unit/v1beta1/test_order.py:210:16: E741 ambiguous variable name 'l'

/cc @crwilcox

--- a/tests/unit/v1/test_order.py
+++ b/tests/unit/v1/test_order.py
@@ -207,8 +207,8 @@ def _double_value(d):
     return encode_value(d)
 
 
-def _int_value(l):
-    return encode_value(l)
+def _int_value(value):
+    return encode_value(value)
 
 
 def _string_value(s):

--- a/tests/unit/v1beta1/test_order.py
+++ b/tests/unit/v1beta1/test_order.py
@@ -207,8 +207,8 @@ def _double_value(d):
     return encode_value(d)
 
 
-def _int_value(l):
-    return encode_value(l)
+def _int_value(value):
+    return encode_value(value)
 
 
 def _string_value(s):

